library/core/resource_loader/src/main/java/org/quiltmc/qsl/resource/loader/api/ResourceLoader.java

```diff
@@ -50,6 +51,24 @@ public interface ResourceLoader {
 		return ResourceLoaderImpl.get(type);
 	}
 
+	/**
+	 * Gets the static resource manager for a given type.
+	 * <p>
+	 * Static resources are different from regular resources in that they
+	 * are available as soon as the manager is requested and are (theoretically) immutable.
+	 * <p>
+	 * It is <i>highly</i> recommended for mods to personally cache data they get from the provided manager in order to
+	 * prevent changes to user-space packs modifying data at runtime.
+	 * <p>
+	 * After the end of the initial boot process, as-in reaching the end of the start process of a server or of the client, the static resource manager may be closed and not re-used.
```

Comment:
> the static resource manager may be closed and not re-used

I don't know what this means. Is it always closed, or just sometimes? Who can/does close it (mods, minecraft, qsl)? Does this mean "may [be closed] and [not re-used]" or "may [be closed and not re-used]"? Please clarify

Reply:
Actually, I'm even more confused; why do we need to close it at all? Why not leave it open indefinitely?

Reply:
Good catch—that looks to be an artifact of an older version of the API. Going to fix this (and even add code to prevent said closures, likely an override of the close method in the static resource manager class) later tonight.

Reply:
Fixed.